api/kogito-api/src/main/java/org/kie/api/marshalling/ObjectMarshallingStrategy.java

```diff
@@ -87,4 +87,8 @@ default String getType(Class<?> clazz) {
         public void write(ObjectOutputStream oos) throws IOException;
     }
 
+	public String marshalToJson(Object value);
```

Comment:
Interface methods are already public. Please apply to the rest of the interface.

```suggestion
	String marshalToJson(Object value);
```